vizro-core/src/vizro/static/css/accordion.css

```diff
@@ -1,39 +1,9 @@
-/* media queries necessary for opening and closing accordion*/
-@media (prefers-reduced-motion: reduce) {
```

Comment:
All of this CSS code is not required anymore as it ships automatically by default with the bootstrap theme